Rakefile

```diff
@@ -45,19 +45,32 @@ end
 
 desc "Fetch all of the data we need from github in a slow way, and write to the cache"
 task :warm_the_cache do
+  puts "Fetching repos..."
   Rake::Task["fetch_repos"].invoke
-  puts "fetched repos"
-  sleep 10
+  puts "Finished fetching repos."
+  puts "Sleeping for 60 seconds to avoid rate-limiting"
+
+  sleep 60
```

Comment:
Might be worth moving this out to a `def avoid_rate_limit` as we use it a few times.

Reply:
I tried that but Rubocop wasn't happy